astropy/io/fits/tests/test_diff.py

```diff
@@ -406,16 +406,17 @@ def test_identical_tables(self):
         c8 = Column("H", format="C", array=[0.0 + 1.0j, 2.0 + 3.0j])
         c9 = Column("I", format="M", array=[4.0 + 5.0j, 6.0 + 7.0j])
         c10 = Column("J", format="PI(2)", array=[[0, 1], [2, 3]])
+        c11 = Column("K", format="QJ(2)", array=[[0, 1], [2, 3]])
```

Comment:
Why does coverage not see this? Hmm.

Reply:
Ah, coverage uploader failed. Ignore the coverage report.

Reply:
codecov reports 33% coverage for the whole project, something went wrong I guess
https://github.com/astropy/astropy/runs/12061184732